tests/test_other.py

```diff
@@ -497,19 +509,19 @@ def test_emcc_7(self):
         # The remainder of this test requires asmjs support
         return
 
-      for opts in [0, 1, 2, 3]:
-        print('mem init in', opts)
-        self.clear()
-        run_process([PYTHON, compiler, path_from_root('tests', 'hello_world.c'), '-s', 'WASM=0', '-O' + str(opts)])
-        if opts >= 2:
-          self.assertTrue(os.path.exists('a.out.js.mem'))
-        else:
-          self.assertFalse(os.path.exists('a.out.js.mem'))
+    for opts in [0, 1, 2, 3]:
+      print('mem init in', opts)
+      self.clear()
+      run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-s', 'WASM=0', '-O' + str(opts)])
```

Comment:
Why is this no longer run with `em++`?

Reply:
There is really no need to run this test in `emcc` and `em++` .  The code is .cpp so `emcc` is autodetecting to build in C++ more.  Running in `em++` doesn't add anything IMHO.